requirements/requirements.in

```diff
@@ -64,7 +64,10 @@ json-delta==2.0
 jsonfield==2.1.1
 jsonobject-couchdbkit~=1.0.1
 jsonobject==0.9.9
-jsonpath-rw==1.4.0
+# jsonpath-ng~=1.5
+# jsonpath-ng with support for "wherenot" operator, and building documents
+# Upgrade when changes merged upstream
+git+git://github.com/kaapstorm/python-jsonpath-rw@wherenot+update_create#egg=jsonpath-ng
```

Comment:
Git dependencies are a headache because pip wants to reinstall them every time it is run. One way to work around this is to build a wheel and publish that on github. We do this for celery:

https://github.com/dimagi/commcare-hq/blob/ddf19c4b792cce0f32e03891700c53b59c16f7a5/requirements/requirements.in#L8

Reply:
Thank you. Good to know! 1f77166